src/lib/containers/table/table-top/DownloadLoginModal.tsx

```diff
@@ -0,0 +1,78 @@
+import { Modal, ModalBody, ModalFooter, Button } from 'react-bootstrap'
+import Login from '../../Login'
+import ModalHeader from 'react-bootstrap/ModalHeader'
+import React, { useState } from 'react'
+
+export type DownloadLoginModalProps = {
+  showModal?: boolean
+  onHide: Function
+}
+
+export const DownloadLoginModal: React.FunctionComponent<DownloadLoginModalProps> = props => {
+  const [showLogin, setShowLogin] = useState(false)
+
+  const getModalContent = (isShowLogin: boolean): JSX.Element => {
+    if (!isShowLogin) {
+      return (
+        <>
+          <p>
+            Anyone can browse public content on the Synapse we site, but in
```

Comment:
Some small typos in this text, here's the fixed version:

Anyone can browse public content on the Synapse website, but in order to download and create content you will need to register for an account using an email address.

Reply:
:woman_facepalming: